indico/modules/events/contributions/models/contributions.py

```diff
@@ -313,6 +313,10 @@ def end_dt(self):
         return self.timetable_entry.start_dt + self.duration if self.timetable_entry else None
 
     @property
+    def end_dt(self):
```

Comment:
this is already in the model; don't add it again ;) (remove this one, i changed it to check `self.timetable_entry` instead of `self.is_scheduled` to avoid possible extra queries)